### DISCUSSION TITLE
Removal of Anti-Cheat

### DISCUSSION THREAD

**Discussion by Vlhss** [YimMenu/YimMenu#548] [General]
Seems like Rockstar has removed most of Arxan from the online gamemode in the newest update on 26 of October. Don't know how to feel about this.

Source: https://www.dexerto.com/gta/gta-online-update-removes-95-of-anti-cheat-and-players-are-baffled-1969410/ and Google

**Comment by SkiddyToast** [comment]
It wasn’t removed, just reworked.

**Reply by Yimura** [reply]
The article bases its contents on one single Tweet.

High quality journalism at its finest.

**Comment by FIying-Scotsman** [comment]
> Seems like Rockstar has removed most of Arxan from the online gamemode in the newest update on 26 of October. Don't know how to feel about this.
> 
> Source: https://www.dexerto.com/gta/gta-online-update-removes-95-of-anti-cheat-and-players-are-baffled-1969410/ and Google

No it wasn't removed completely. They've reworked some _parts_ of it (the sig scanner for example) and supposedly removed Arxan from some methods which were protected previously (and added it to previously unprotected functions/methods) but it's still there.

I think more interestingly they've reduced the amount of things generating metrics while adding a couple new ones. Your client barely sends out metrics now (unlike before where it was a torrent of telemetry depending on what you were doing) but that may change over the coming weeks as something similar happened when Arxan was first introduced and then turned on a week later.

**Comment by ghost** [comment]
TezFunz pulling numbers out of his ass per usual.

Sometimes I wonder why people listen to that bloke.
